experiments/tf_trainer/tf_gru_attention/model.py

```diff
@@ -31,7 +32,7 @@
 
 class TFRNNModel(base_model.BaseModel):
 
-  def __init__(self, text_feature_name, target_labels):
+  def __init__(self, text_feature_name: str, target_labels: List[str]) -> None:
```

Comment:
Here you suggest the type of target_labels should be a list, but in run.py you call it with a dict.

Reply:
Let me fix that... mypy is failing me